### DISCUSSION TITLE
Is the minimum Java runtime version for KC16+ still 8?

### DISCUSSION THREAD

**Discussion by dasniko** [keycloak/keycloak#9308] [Q&A]
When working with KC16+, I started to get some strange behaviors during my extension tests when using Java 8. I got rid off the exceptions and errors when using Java 11.
In the first shot, I thought I did something wrong, but now there's also a mailinglist entry with similar experience: https://groups.google.com/g/keycloak-user/c/-4554RE3-8M

I don't have more proper details on that currently, but maybe it is worth investigating!?
Perhaps I can deliver some more behavior and error messages in the next few days.

**Comment by stianst** [comment]
We haven't dropped Java 8 support for the server yet, but I would expect us to do that really soon (see https://github.com/keycloak/keycloak/issues/9310).

**Comment by shawnweeks** [comment]
I'd say not supporting LDAP on Java 8 is a pretty breaking change so even though we say we support Java 8 things will be broken if you attempt to run Keycloak on Java 8
